src/GithubMergeTool/GithubMergeTool.cs

```diff
@@ -180,9 +182,40 @@ git commit
             jsonBody = JObject.Parse(await response.Content.ReadAsStringAsync());
 
             var prNumber = (string)jsonBody["number"];
+            var mergeable = (bool?)jsonBody["mergeable"];
+            if (mergeable == null)
+            {
+                const int maxAttempts = 5;
+                var attempt = 0;
+
+                Console.Write("Waiting for mergeable status");
+                while (mergeable == null && attempt < maxAttempts)
```

Comment:
consider logging an error/warning if attempt exceeds maxAttempts.  I would guess the mergeable attribute should always eventually be computed (either false or true).